docroot/themes/contrib/civictheme/civictheme_library/components/02-molecules/logo/logo.stories.js

```diff
@@ -18,21 +18,52 @@ export const Logo = (knobTab) => {
       Light: 'light',
       Dark: 'dark',
     }, 'light', generalKnobTab),
+    type: radios('Type', {
+      Default: 'default',
+      Inline: 'inline',
+      Stacked: 'stacked',
+    }, 'default', generalKnobTab),
+    with_secondary_image: boolean('With secondary image', false, generalKnobTab),
     logos: {},
     url: text('Link', randomUrl(), generalKnobTab),
     title: text('Title', 'Logo title', generalKnobTab),
     attributes: text('Additional attributes', '', generalKnobTab),
     modifier_class: `civictheme-logo-example story-wrapper-size--small ${text('Additional class', '', generalKnobTab)}`,
   };
 
-  generalKnobs.logos = {
+  generalKnobs.logos = generalKnobs.with_secondary_image ? {
     mobile: {
-      src: LOGOS[generalKnobs.theme].mobile,
-      alt: 'Logo mobile alt text',
+      primary: {
+        src: LOGOS[generalKnobs.theme].mobile.primary,
```

Comment:
please reverse this
```
LOGOS[generalKnobs.theme].primary.mobile
```

Reply:
By this it means 
 * - logos: [object] Object with 'primary' and 'secondary' keys:
 *    - mobile:
 *       - src: [string] URL for the logo.
 *       - alt: [string] Image alt text.
 *    - desktop:
 *       - src: [string] URL for the logo.
 *       - alt: [string] Image alt text.

Reply:
```
{
  "primary": {
    "mobile" : {
      "src": "",
      "alt": "",
    },
    "desktop" : {
      "src": "",
      "alt": "",
    }
  },
  "secondary": {
    "mobile" : {
      "src": "",
      "alt": "",
    },
    "desktop" : {
      "src": "",
      "alt": "",
    }
  }  
}
```

Reply:
Updated to this above format 
But I feel the loop should go through screen mode and then the logotype 
The ability will result in 
<mobile><primary></primary></mobile>
<desktop><primary></primary></desktop>
<mobile><secondary></secondary></mobile>
<desktop><secondary></secondary></desktop>
![Screenshot 2022-09-09 at 12 22 48 PM](https://user-images.githubusercontent.com/83997348/189289576-6b3366d5-2e15-468a-aa83-5a5776142f50.png)


As to loop get a logo with a common link 
<mobile><primary></primary><secondary></secondary></mobile>
<desktop><primary></primary><secondary></secondary></desktop>

Reply:
@joshua-salsadigital 
I agree - all elements of the logo at a specific breakpoint should be rendered together.